apollo-router/src/plugins/telemetry/metrics/apollo/duration_histogram.rs

```diff
@@ -1,13 +1,14 @@
+use std::ops::AddAssign;
 use std::time::Duration;
 
 use serde::Serialize;
 #[derive(Serialize, Debug)]
-pub(crate) struct DurationHistogram {
-    pub(crate) buckets: Vec<i64>,
-    pub(crate) entries: u64,
```

Comment:
Do you understand why these field differed in their signedness before (which this PR is changing)? (I don't, just thought it was a notable change and not mentioned in the PR description.

(Maybe it was just that we originally thought we might want to do the run-length-encoding that we don't do in Router?)

Then it looks like most of the callers that stay integral use i64... seems like u64 should be fine?

Reply:
The Router does contain an implementation of length-encoding runs of zeroes as negative values, but it looks like it’s only used in tests. For… testing itself. See `fn to_array` near the end of this file.

Should we change Router to actually use this?

Reply:
I don't have a strong feeling about whether or not it's useful, but there is a comment in Router that makes a persuasive argument that it's not worth the tradeoff here. (Not sure if the performance issues have been validated or not.) If you're not using it for now might as well use u64s for now?

Reply:
Done